models/_sector/rebase_events/_schema.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+
+models:
+  - name: rebase_events
+    meta:
+      blockchain: ethereum
+      sector: rebase
+      contributors: hildobby
+    config:
+      tags: ['rebase', 'ethereum']
+    description: >
+        ERC20 rebase events
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
```

Comment:
to be safe, do you think we should proactively add `blockchain` to unique keys in model config and then test here, assuming you add future chains?

Reply:
yeah sure